src/Microsoft.xunit.extensibility.execution.netcore/Microsoft.xunit.extensibility.execution.netcore.depproj

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
```

Comment:
This project is being consumed in Microsoft/xunit-performance, to package their dependencies.